fishtest/fishtest/views.py

```diff
@@ -324,18 +327,24 @@ def validate_form(request):
     'new_options' : request.POST['new-options'],
     'username' : authenticated_userid(request),
     'tests_repo' : request.POST['tests-repo'],
+    'info' : request.POST['run-info']
   }
 
-  # Fill new_signature from commit info if left blank
-  if len(data['new_signature']) == 0:
-    found = False
+  # Fill new_signature/info from commit info if left blank
+  if len(data['new_signature']) == 0 or len(data['info']) == 0:
     api_url = data['tests_repo'].replace('https://github.com', 'https://api.github.com/repos')
     api_url += ('/commits' + '/' + data['new_tag'])
-    bs = re.compile('(^|\s)[Bb]ench[ :]+([0-9]{7})', re.MULTILINE)
     c= requests.get(api_url).json()
-    m = bs.search(c['commit']['message'])
-    if m:
-      data['new_signature']= m.group(2)
+    if not 'commit' in c:
+      raise Exception('Cannot find branch in developer repository')
+    if len(data['new_signature']) == 0:
+      bs = re.compile('(^|\s)[Bb]ench[ :]+([0-9]{7})', re.MULTILINE)
+      m = bs.search(c['commit']['message'])
+      if m:
+        data['new_signature']= m.group(2)
+    if len(data['info']) == 0:
+        data['info'] = ('STC: ' if re.match('^[012][0-9][^0-9].*', data['tc']) else 'LTC: ') \
```

Comment:
The 'STC: ' strings are a little bit hurting the readability of the test comments, while bringing no information (less than 1 bit of information on average :-))

Maybe this would be sufficient:

````
data['info'] = ('' if re.match('^[012][0-9][^0-9].*', data['tc']) else 'LTC: ') \
````

Reply:
I agree...